PGM/src/main/java/tc/oc/pgm/tablist/MatchFooterTabEntry.java

```diff
@@ -24,6 +27,7 @@ public MatchFooterTabEntry get(Match key) {
             return new MatchFooterTabEntry(key);
         }
     }
+    @Inject private static LocalServerDocument document;
```

Comment:
Please test this, and make sure the injection works. I think I tried this before but something went wrong.

Reply:
The injection works. You probably didn't request the static injection(I think)